packages/bpk-component-breadcrumb/src/BpkBreadcrumb.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './BpkBreadcrumb.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkBreadcrumb = props => {
+  const { children } = props;
+
+  const className = getClassName('bpk-breadcrumb');
+
+  return <div className={className}>{children}</div>;
+};
+
+BpkBreadcrumb.defaultProps = {
+  children: null,
+};
+
+BpkBreadcrumb.propTypes = {
+  children: PropTypes.node,
```

Comment:
I think this should be required. It doesn't make sense to have a breadcrumb without children